src/ContainerCommitModal.jsx

```diff
@@ -118,11 +128,11 @@ const ContainerCommitModal = ({ container, localImages }) => {
             <FormGroup fieldId="commit-dialog-pause" label={_("Options")} isStack hasNoPaddingTop>
                 <Checkbox id="commit-dialog-pause"
                           isChecked={pause}
-                          onChange={setPause}
+                          onChange={(_, val) => setPause(val)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1266-20230413-123832-6bec41be-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)